net-p2p/cardano-node/files/cardano_node.in

```diff
@@ -57,18 +70,30 @@ extra_commands="status"
 
 load_rc_config $name
 : ${cardano_node_enable:=NO}
+: ${cardano_node_user:="%%USERS%%"}
+: ${cardano_node_group:="%%GROUPS%%"}
+: ${cardano_node_net:=mainnet}
 : ${cardano_node_home:="/var/db/cardano_node"}
 : ${cardano_node_db:="${cardano_node_home}/db"}
+: ${cardano_node_cfg_dir:="${cardano_node_home}/conf"}
+: ${cardano_node_logs_dir:="${cardano_node_home}/logs"}
 : ${cardano_node_host:="0.0.0.0"}
 : ${cardano_node_port:="6000"}
 : ${cardano_node_socket:="${cardano_node_home}/cardano-node.sock"}
-: ${cardano_node_topology:="mainnet-topology.json"}
-: ${cardano_node_config:="mainnet-config.json"}
+: ${cardano_node_topology:="${cardano_node_cfg_dir}/${cardano_node_net}-topology.json"}
+: ${cardano_node_config:="${cardano_node_cfg_dir}/${cardano_node_net}-config.json"}
 : ${cardano_node_rts_flags:="-N -A64m -n4m -F1.2 -qg1"}
 : ${cardano_node_flags:=""}
 
-topology_file="${cardano_node_home}/testnet-topology.json"
-config_file="${cardano_node_home}/testnet-config.json"
+# set up dependant variables
+procname="${command}"
+required_files="${cardano_node_config}"
+apply_cmd="/usr/bin/apply"
+fetch_cmd="/usr/bin/fetch -aq -o ${cardano_node_cfg_dir}"
+url="https://hydra.iohk.io/job/Cardano/cardano-node/cardano-deployment/latest-finished/download/1"
+latest_files="config byron-genesis shelley-genesis alonzo-genesis topology"
```

Comment:
Can you explain this line? I though, that the daemon only needs a config and a topology file. These files do not get passed to the `cardano_node` call, how are they used?